guides/development-workflow.md

```diff
@@ -448,6 +514,9 @@ have chosen the following existing style guides for our work:
   [Sass Guidelines - Syntax and formatting](https://sass-guidelin.es/#syntax--formatting)
 - JavaScript - [Standard](https://standardjs.com/)
 
+When joining an existing code base we suggest following any existing code style
```

Comment:
I'd suggest removing the specific code standards from this guide because they'll end up out of date (we use Prettier for JS for example).

Reply:
I think its useful to point out our defaults. Though they should be up to date :)